src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -534,6 +540,28 @@ describe('lib/viewers/BaseViewer', () => {
             expect(base.disableAnnotationControls).toBeCalled();
             expect(base.processAnnotationModeChange).toBeCalledWith(AnnotationMode.NONE);
         });
+
+        test('should enable the focus trap', () => {
+            jest.spyOn(FocusTrap.prototype, 'constructor');
```

Comment:
It's a little unusual to spy on the prototype. Can we instead make sure that `base.focusTrap` is defined, then  use `jest.spyOn(base.focusTrap, 'enable')`? That way we're testing the instance rather than the prototype.

Reply:
Yeah, I don't prefer spying the prototype but I went this route because I wanted to test the case where we actually create the FocusTrap vs reuse it. I went with the approach you suggested below when reusing the existing focus trap